masterlist.yaml

```
@@ -26919,3 +26944,4 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0xE405E000
         util: 'SSEEdit v4.0.4c'
+        
```

The newline has extra whitespace as well:
```suggestion
        util: 'SSEEdit v4.0.4c'

```